src/lib/mina_base/test/zkapp_command/call_forest.ml

```diff
@@ -0,0 +1,320 @@
+open Core_kernel
+open Signature_lib
+open Mina_base
+open Mina_base.Zkapp_command.Call_forest
+
+(* TODO: move generators to the library once the code isn't actively being
+     worked on *)
+module Tree = struct
+  include Tree
+
+  module Stable = struct
```

Comment:
Since we only want to test with the latest type, I think it's better not to mention the `Stable` hierarchy here.

I think this will work:
```
module Tree = struct
  include Tree

  let gen ... = ...
end
```

Reply:
Should we leave `quickcheck_generator` as an alias for `gen` (as in `account_update.ml`) to allow the usage of `ppx_quickcheck`?

Reply:
Since type definitions mention specific versions of types, could it be possible that, for a same type, different versions are used throughout the code? For example, having a type definition somewhere using `Account_update.Stable.V1.t` and a different one using `Account_update.Stable.V2.t`. If that's true, couldn't that create an issue with the generator being defined only for the latest version of that type, since both types are possibly incompatible and you couldn't use the generator for creating values that are still based on the previous version?

Reply:
The convention, usually enforced by the `%%versioned` ppx, is that `Foo.t` is the same as `Foo.Stable.Latest.t`, which is the same as `Foo.Stable.Vn.t`, for some `n`. 

Older versions are kept around only for coercion to the latest type, in a few limited circumstances.

If you had a type definition like:
```
module V1 = struct
  type t = Foo of Bar.Stable.V1.t
end
```
Then the ppx generates
```
type t = Foo of Bar.t
```
If `Bar.Stable.V1.t` is not the latest, you'll get a type error.

Reply:
So I'm still requesting this change.

Reply:
Thanks for the explanation. I wasn't totally aware of how `ppx_version` worked.